src/components/RecipeLinks.astro

```diff
@@ -0,0 +1,76 @@
+---
+import { getEntryBySlug } from 'astro:content';
+
+export interface Props {
+	slugs: string[];
+}
+
+function convertSlug(slug: string) {
+	let corrected = slug;
+	if (slug.startsWith('/')) {
+		corrected = corrected.slice(1);
+	}
+	if (slug.endsWith('/')) {
+		corrected = corrected.slice(0, -1);
+	}
+	return corrected;
+}
+
+const slugs = Astro.props.slugs.map(convertSlug);
+
+const entries = await Promise.all(
+	slugs.map(async (slug, i) => {
+		let entry = await getEntryBySlug('docs', slug);
+		if (!entry) throw Error(`Could not find entry for slug "${Astro.props.slugs[i]}"`);
+		return entry;
+	})
+);
+
+const isList = entries.length > 1;
+const noun = isList ? 'Recipes' : 'Recipe';
```

Comment:
```suggestion
const noun = isList ? 'recipes' : 'recipe';
```
I saw you changed this intentionally, but are we sure about the case here? Recipe isn't a proper noun for us, so I think lowercase is more consistent.